.devcontainer/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# Stage 1: Install Node.js
+FROM node:20 as node-stage
+
+# Verify Node.js installation
+RUN node -v
+RUN npm -v
+RUN corepack enable
+RUN yarn --version
+
+# Stage 2: Install Rust and Nextest
+FROM docker.io/rust:latest as rust-stage
+
+# Install cargo-nextest
+RUN cargo install cargo-nextest
```

Comment:
I got this: `Error: there is no registered task type 'cargo'. Did you miss installing an extension that provides a corresponding task provider?`

Reply:
Huh...   what's going on with this @dorianvp ?  Do you also get that error?